### DISCUSSION TITLE
Question About Target Rate Sensors

### DISCUSSION THREAD

**Discussion by StephenMilner** [BottlecapDave/HomeAssistant-OctopusEnergy#171] [Q&A] [answered]
I set up a target rate sensor to charge the car

2.5 hours
Intermittent
01:30
05:30
False

Two nights ago it worked well. 

Yesterday I added an offset of -00:02:30 by configuring the sensor

At 19:30 I had to update my HA docker.

Last night my rate sensor did not trigger at all. 

Could the upgrade have lost my forward 24h rates from Octopus?

Name
overnight_car_charge
Hours
2.5
Type
Intermittent 
MPAN 
XXXXXX
Rolling target
false
Start time
01:30
End time
05:30
Offset
-00:02:30
Is target export
false
Target times
Next time
—
Current duration in hours
0
Next duration in hours
0

**Comment by BottlecapDave** [comment, marked as answer]
Hello, sorry for the late response. 

There was a [bug issued](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/issues/173) around offsets and target rate sensors which was fixed in a recent release which could have caused this issue. Have you updated to the latest release and tried again?

**Reply by StephenMilner** [reply]
Thanks.

I have and it's sorted now.
